pkg/account/downloader/internal/http/client.go

```diff
@@ -53,7 +54,7 @@ func (c *Client) GetGroupsForUser(ctx context.Context, userEmail string, account
 }
 
 func (c *Client) GetPolicies(ctx context.Context, account string) ([]accountmanagement.PolicyOverview, error) {
-	r, resp, err := c.PolicyManagementAPI.GetPolicyOverviewList(ctx, "account", account).Execute()
+	r, resp, err := c.PolicyManagementAPI.GetPolicyOverviewList(ctx, account, "account").Execute()
```

Comment:
Can we move "account" into a constant?

Reply:
We could, along also e.g. with `"environment"`, but we must decide where these should go. I am also not convinced of the benefit of maintaining separate clients for download and deployment. Ideally we should follow this PR with one consisting of improvements, I think @jskelin had something in the works.